internal/version/version.go

```diff
@@ -7,7 +7,7 @@ import (
 
 var version = ""
 
-func GetVersion() string {
+func GetVersion(withSum bool) string {
```

Comment:
If following the previous suggestion, I think I would also prefer two separate functions. This feels more Go-like and readable in calling code to me than parameter flags:

```go
func GetVersion() string {
    return version
}

func GetVersionWithSum() string {
    return fmt.Sprintf("%s (%s)", version, sum)
}
```

Reply:
I had the same question when doing this.
Good to me :)